reference-controllers/zookeeper-controller/src/main.rs

```diff
@@ -305,10 +308,7 @@ async fn reconcile_zk_node(zk: &ZookeeperCluster, client: Client) -> Result<(),
                 zk_client
                     .create(
                         "/zookeeper-operator",
-                        format!("CLUSTER_SIZE={}", zk.spec.replicas)
-                            .as_str()
-                            .as_bytes()
-                            .to_vec(),
+                        "".as_bytes().to_vec(),
```

Comment:
Why not specify the cluster size when creating the node?

Reply:
I think this data is meaningless. The official one leaves it as blank so I did so. The CLUSTER_SIZE is stored in the next layer(/zookeeper-operator/$zk-name)